backend/src/data/migrations/20220830154032_add_chat_messages_table.ts

```diff
@@ -0,0 +1,52 @@
+import { Knex } from 'knex';
+
+enum TableName {
+  CHAT_MESSAGES = 'chat_messages',
+  USERS = 'users',
+}
+
+enum ColumnName {
+  ID = 'id',
+  CREATED_AT = 'created_at',
+  UPDATED_AT = 'updated_at',
+  SENDER_ID = 'sender_id',
+  RECEIVER_ID = 'receiver_id',
+  CHAT_ID = 'chat_id',
+  MESSAGE = 'message',
+}
+
+const DELETE_STRATEGY = 'CASCADE';
+
+async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable(TableName.CHAT_MESSAGES, (table) => {
+    table.increments(ColumnName.ID).primary();
+    table
+      .dateTime(ColumnName.CREATED_AT)
+      .notNullable()
+      .defaultTo(knex.fn.now());
+    table
+      .dateTime(ColumnName.UPDATED_AT)
+      .notNullable()
+      .defaultTo(knex.fn.now());
+    table
+      .integer(ColumnName.SENDER_ID)
+      .references(ColumnName.ID)
+      .inTable(TableName.USERS)
+      .notNullable()
+      .onDelete(DELETE_STRATEGY);
+    table
+      .integer(ColumnName.RECEIVER_ID)
+      .references(ColumnName.ID)
+      .inTable(TableName.USERS)
+      .notNullable()
+      .onDelete(DELETE_STRATEGY);
+    table.text(ColumnName.MESSAGE).notNullable();
+    table.string(ColumnName.CHAT_ID).notNullable();
```

Comment:
```suggestion
    table.uuid(ColumnName.CHAT_ID).notNullable();
```
can we use uuid type?